modules/serverless-create-func-kn.adoc

```diff
@@ -6,6 +10,12 @@ You can create a basic serverless function using the `kn` CLI.
 
 You can specify the path, runtime, template, and repository with the template as flags on the command line, or use the `-c` flag to start the interactive experience in the terminal.
 
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on the cluster.
+* You have installed the `kn` CLI.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
```

Comment:
Since we are noting prerequisites for each individual command, this is not actually a requirement for `create` - only for `deploy`.
```suggestion
```

Reply:
@lance I think here it refers to openshift project (.i.e `oc new-project`, `oc project`) so a namespace you have access to.